docs/conf.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*-
+#
+# MPAS-Model documentation build configuration file, created by
+# sphinx-quickstart on Sat Mar 25 14:39:11 2017.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+import os
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['sphinx.ext.autodoc',
+              'sphinx.ext.autosummary',
+              'sphinx.ext.intersphinx',
+              'sphinx.ext.mathjax',
+              'sphinx.ext.viewcode',
+              'sphinx.ext.napoleon']
+
+autosummary_generate = True
+
+# Otherwise, the Return parameter list looks different from the Parameters list
+napoleon_use_rtype = False
+# Otherwise, the Attributes parameter list looks different from the Parameters
+# list
+napoleon_use_ivar = True
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+source_suffix = ['.rst']
+# source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'MPAS-Model'
+copyright = u'Copyright (c) 2013-2020,  Los Alamos National Security, LLC (LANS) (Ocean: LA-CC-13-047;' \
+            u'Land Ice: LA-CC-13-117) and the University Corporation for Atmospheric Research (UCAR).'
+author = u'Xylar Asay-Davis, Doug Jacobsen, Michael Duda, Mark Petersen, ' \
+         u'Matt Hoffman, Adridan Turner, Philip Wolfram'
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+if 'DOCS_VERSION' in os.environ:
+    version = os.environ.get('DOCS_VERSION')
+    release = version
+else:
+    # The short X.Y.Z version.
+    version = '7.0'
+    # The full version, including alpha/beta/rc tags.
+    release = '7.0'
```

Comment:
It would be nice to parse these from somewhere.  @mgduda, would it be safe to parse this from the top of the README.md?  Or is there some other "official" place?